frame/nfts/src/features/attributes.rs

```diff
@@ -159,6 +159,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 				Collection::<T, I>::get(&collection).ok_or(Error::<T, I>::UnknownCollection)?;
 
 			if let Some(check_owner) = &maybe_check_owner {
+				// deposit.account.is_none(); maybe_check_owner.is_some()
```

Comment:
Why commented out code?

Reply:
It's not a code, just a note to understand the code below better :)

Reply:
ah that wasn't clear. I thought it was code for what was to come.

Reply:
changed